sampleresources.md

```diff
@@ -0,0 +1,15 @@
+#Helpful sample resorces for Microsoft ASP.NET WebHooks
+Please see the initial [documentation](http://docs.asp.net/projects/aspnetwebhooks/en/latest/) for details.
+
+Following(s) are helpful resources from our contributors: 
+
+##Samples
+- [GitHub](https://github.com/aspnet/WebHooks/tree/master/samples/#)
```

Comment:
These URLs will be automatically fixed, once current dev branch merged down to master.